share/completions/gacutil.fish

```diff
@@ -0,0 +1,36 @@
+function __fish_seen_tokens -d 'Check whether there are already specified tokens in command line buffer'
+  set -l cmd (commandline -oc)
```

Comment:
this has a subtle bug because it's missing `--current-process`
If your command line is `echo sometok; gacutil `, then `__fish_seen_tokens sometok` is true.
That's why most places use `commandline -opc`.